src/main/java/io/mosip/iiitb/odte/lib/MessageBrokerWrapperConfig.java

```diff
@@ -0,0 +1,2 @@
+package io.mosip.iiitb.odte.lib;
+
```

Comment:
??

Reply:
can you please provide some more clarification